Randomizer/Patches/IPSPatcher.py

```diff
@@ -0,0 +1,45 @@
+from pathlib import Path
+from BaseClasses.PatchBase import *
+
+class IPSPatcher(PatchBase):
+    
+    def __init__(self, file, params = None):
+        super().__init__(file, params)        
+        
+    def __GetFile(self):
+        self.IPS_Folder = Path("IPSFiles/")
+        self.PatchFile = self.IPS_Folder / self.params
+        self.IPSFile = open(self.PatchFile, "r+b")
+    
+    def __Write(self):
+        Offset = 0
+        Size = 0
+        RLE_Size = 0
+        Value = 0
```

Comment:
please consider renaming those that are intended to be locally used with lower case or something

Reply:
Lowercase for locals sounds good to me. I'll make those changes and implement that style going forward 🙂

Reply:
Fixed under the local variable styling commit

Reply:
I was feeling productive and gave the IPS patch script another clean up today. Removed all of the unnecessary variable calling, fixed some casing I missed, and also added some clarifying comments.